lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -1002,4 +1036,17 @@ private CriResponseItem createCriErrorResponseStoreItem(Instant dateCreated) {
         criResponseItem.setStatus(CriResponseService.STATUS_ERROR);
         return criResponseItem;
     }
+
+    private static SignedJWT createOperationalProfileVc(Vot vot) throws Exception {
```

Comment:
Do you think this should go into TestFixtures (alongside other JWT creation methods)?

Might also overlap with @TalatAydinGDS 's work

Reply:
I don’t think so. Maybe a more fully fledged version but these are very much MVP’s to test the functionality